src/HT6.java

```diff
@@ -0,0 +1,32 @@
+
+import java.util.ArrayList;
+import java.util.HashSet;
+
+public class HT6 {
+    public HashSet findPairsOfLitera(String [] arr){
+        ArrayList<String> pairedWords = new ArrayList<String>(  );
+        HashSet<Character> resultList = new HashSet<Character>(  );
+        for (int i = 0; i<arr.length; i++){
+            char[] charArray = arr[i].toCharArray();
+            HashSet <Character> charsList = new HashSet<Character>();
+            for( int j = 0; j<charArray.length;j++){
+                if (!charsList.contains(charArray[j])){
+                    charsList.add(charArray[j]);
+                }
+            }
+            if ((arr[i].length()%charsList.size()==0) && (arr[i].length()>charsList.size())){
```

Comment:
Wrong condition here, add word `abacab` to array and check
Also as said in task you need to put only first two words